mava/networks.py

```diff
@@ -16,13 +16,16 @@
 from typing import Callable, Dict, Sequence, Tuple, Union
 
 import chex
-import distrax
 import jax
 import jax.numpy as jnp
 import numpy as np
+import tensorflow_probability.substrates.jax.distributions as tfd
 from flax import linen as nn
 from flax.linen.initializers import orthogonal
+from jax.experimental import checkify
```

Comment:
Similiar to the other comment. But we should please not be using anything from `jax.experimental`